src/dom.js

```diff
@@ -0,0 +1,51 @@
+const dom = () => {
+  const cont = document.getElementById('cont');
+  const buttonF = document.getElementById('changeF');
+  const buttonC = document.getElementById('changeC');
+  const temp = document.getElementById('temperature');
+  const city = document.getElementById('city');
+  const summary = document.getElementById('summary');
+  const icon = document.getElementById('icon');
+  const form = document.getElementById('form');
+  const error = document.getElementById('error');
+
+  const btn = document.getElementById('button');
+
+  btn.addEventListener('click', (e) => {
+    e.preventDefault();
+
+    const search = document.getElementById('search').value;
+
+    const api = `https://api.openweathermap.org/data/2.5/weather?q=${search}&units=metric&appid=96167910369f1228a67e094ac8e334b6`;
+
+    fetch(api, { mode: 'cors' })
+      .then(response => response.json())
+      .then(item => {
+        error.style.display = 'none';
+        cont.style.display = 'flex';
+        temp.textContent = `${item.main.temp} C°`;
+        summary.textContent = item.weather[0].description.charAt(0).toUpperCase() + item.weather[0].description.slice(1);
+        city.textContent = item.name;
+        icon.src = `http://openweathermap.org/img/wn/${item.weather[0].icon}@2x.png`;
+
+        buttonF.addEventListener('click', () => {
+          temp.textContent = `${((parseFloat(item.main.temp) * (9 / 5)) + 32).toFixed(2)} F°`;
+          buttonF.style.display = 'none';
+          buttonC.style.display = 'block';
+        });
+
+        buttonC.addEventListener('click', () => {
+          temp.textContent = `${item.main.temp} C°`;
+          buttonF.style.display = 'block';
+          buttonC.style.display = 'none';
+        });
+      })
+      .catch(() => {
```

Comment:
Let's separate the DOM from application logic, you can create another file for DOM.

In your case, we have DOM task inside the API logic.

![image](https://user-images.githubusercontent.com/31897434/73420341-70e18b80-4322-11ea-840a-fa14e04b592b.png)

https://microverse.pathwright.com/library/javascript-module-v10-copy/166786/path/step/97635172/

Reply:
Thanks for the review! 👍🏼